plugins/HAA4bAnalysis.cc

```diff
@@ -69,6 +72,11 @@ class HAA4bAnalysis : public edm::EDAnalyzer {
   std::string bdiscr_;
   double minPt1_;
   double minPt4_;
+  bool runningOnData_;
```

Comment:
But this is set to false by default. Shouldn't you add a line in multicrab_data_HAA4bAnalysis.py
to turn this on? (I can do it after your next commit BTW)